definitions/features/downstream.rb

```diff
@@ -23,13 +23,25 @@ def setup_repositories(version)
       execute!(%(subscription-manager register #{org_options}\
                   --activationkey #{shellescape(activation_key)} --force))
     else
-      execute!(%(subscription-manager refresh))
       execute!(%(subscription-manager repos --disable '*'))
       enable_options = rh_repos(version).map { |r| "--enable=#{r}" }.join(' ')
       execute!(%(subscription-manager repos #{enable_options}))
     end
   end
 
+  def absent_repos(version)
+    all_repo_lines = execute!(%(LANG=en_US.utf-8 subscription-manager repos --list | ) +
```

Comment:
@iNecas 
```
--------------------------------------------------------------------------------
Validate availability of repositories:
- Validating availability of repositories for 6.3                     [FAIL]
Failed executing LANG=en_US.utf-8 subscription-manager repos --list | grep '^Repo ID:', exit status 1
--------------------------------------------------------------------------------
Scenario [Checks before upgrading to Satellite 6.3.z] failed.
```

Reply:
Thanks @ntkathole for catching this, updated the code